app/models/foreman_docker/docker.rb

```diff
@@ -1,6 +1,7 @@
 require 'uri'
 
 module ForemanDocker
+  # rubocop:disable Metrics/ClassLength
```

Comment:
100 is too small a number for a class line length IMO. I was having issues breaking this into multiple modules and thought may be either u or @eLobato can suggest approaches.